dispatch.yaml

```diff
@@ -14,11 +14,5 @@ dispatch:
 - url: "*/tasks/email-subscribers"
   service: notifier
 
-- url: "*/tasks/outbound-email"
-  service: default
-
-- url: "*/py2"
-  service: default
-
 - url: "*/*"
-  service: app-py3
+  service: default
```

Comment:
Remove the comments from line 3 to line 8

Reply:
I shortened the comments, but I still wanted something in there to explain that inbound email goes to the default service.